src/cts/src/HTreeBuilder.h

```diff
@@ -397,6 +399,8 @@ class HTreeBuilder : public TreeBuilder
   unsigned minLengthSinkRegion_ = 0;
   unsigned clockTreeMaxDepth_ = 0;
   static constexpr int min_clustering_sinks_ = 200;
+  std::vector<unsigned> clusterDiameters_ = {50, 100, 200};
```

Comment:
I would make these more tech dependent.  Something like N * buffer_height might work better.  We can merge this and revisit it.